buf.work.yaml

```
@@ -1,4 +1,5 @@
 version: v1
 directories:
   - proto
-  - third_party/proto
\ No newline at end of file
+  - third_party/proto
+  - legacy_protos
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)